dev.zed.Zed.metainfo.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.zed.Zed</id>
+  <name>Zed</name>
+  <summary>High-performance code editor</summary>
+  <developer id="dev.zed">
+    <name translate="no">Zed Industries, Inc.</name>
+  </developer>
+  <description>
+    <p>
+      *NOTE*: This is the official release of Zed, packaged into a Flatpak by the community. This repackaging is not yet supported by Zed Industries, Inc.
```

Comment:
That's totally fine and expected -- since [ConradIrwin from Zed highlights they're comfortable with keeping it community maintained for now](https://github.com/flathub/flathub/pull/5253#issuecomment-2248470316)